src/main/java/net/fabricmc/loader/impl/launch/knot/KnotClassLoader.java

```diff
@@ -51,7 +51,7 @@ public void addURL(URL url) {
 	private final KnotClassDelegate<KnotClassLoader> delegate;
 
 	KnotClassLoader(boolean isDevelopment, EnvType envType, GameProvider provider) {
-		super(new DynamicURLClassLoader(new URL[0]));
+		super("knot", new DynamicURLClassLoader(new URL[0]));
```

Comment:
Perhaps something like 'fabric-knot' would be more clear.

It would be nice to encourage a naming convention that makes it clear which classloader is being specified, AND where it comes from.